File: 2term.labs/lab 2.2/c++/Circle.cpp
#include "Circle.h"
#include <iostream>
#include <cmath>
#define PI 3.141592653589793

using namespace std;

Circle::Circle(double radius) : _radius(radius) {};

double Circle::getRadius() {
	return _radius;
}

double Circle::getArea() const {
	double area = PI * pow(_radius, 2);
	return area;
}

double Circle::getPerimeter() const {
	double perimeter = 2 * PI * _radius;
	return perimeter;
}

File: 2term.labs/lab 2.2/c#/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary
{
    public class Circle : Shape
    {
        public double Radius { get; private set; }
        public Circle(double radius)
        {
            Radius = radius;
        }

        public override double GetArea()
        {
            double area = Math.PI * Radius * Radius;
            return area;
        }

        public override double GetPerimeter()
        {
            double perimeter = Math.PI * Radius * 2;
            return perimeter;
        }
    }
}
